.gitlab/container_build/docker_linux.yml

```diff
@@ -127,6 +127,42 @@ docker_build_agent7:
     TEST_IMG: "true"
     BUILD_ARG: --target release --build-arg PYTHON_VERSION=3 --build-arg DD_AGENT_ARTIFACT=datadog-agent_7*_amd64.deb
 
+# NOTE this is a temporary workaround and the method taken in #14438 is
+# preferred in the long-term.
+docker_build_agent7_single_machine_performance:
```

Comment:
You need to use explicit `stage`, `image`, `tags` on the job if you're not inheriting from any of the base jobs in this file.

Something like this should do the trick:

```
  stage: container_build
  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64:$DATADOG_AGENT_BUILDIMAGES
  tags: ["runner:docker"]
```

Reply:
Addressed in [dbd866f](https://github.com/DataDog/datadog-agent/pull/14477/commits/dbd866f0dd2e6a620d11f3ece2ec27664ad49c42)